service/src/main/java/bio/terra/cbas/controllers/RunSetsApiController.java

```diff
@@ -307,9 +305,8 @@ public ResponseEntity<RunSetStateResponse> postRunSet(RunSetRequest request) {
 
   @Override
   public ResponseEntity<AbortRunSetResponse> abortRunSet(UUID runSetId) {
-    // check if current user has compute permissions on the workspace
-    if (!samService.hasComputePermission()) {
-      throw new ForbiddenException(SamService.COMPUTE_ACTION, SamService.RESOURCE_TYPE_WORKSPACE);
+    if (!samService.hasWritePermission()) {
```

Comment:
TOL abort is interesting... I probably wouldn't want to start a new app just to abort a workflow, but if an app _is_ already running I don't see too much harm in sending the request along